src/styles/blocks/snake.scss

```
@@ -0,0 +1,24 @@
+.snake {
+  width: 100%;
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax($block-size, 1fr));
+
+  @media(min-width: 1200px) {
+    & {
+      grid-template-columns: repeat(3, minmax($block-size, 1fr));
+    }
+  }
```

```suggestion
  @media(min-width: 1200px) {
      grid-template-columns: repeat(3, minmax($block-size, 1fr));
  }
```
You can use this styles here without selector)

---

> You can use this styles here without selector)

Good point! Thanks!)